pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -167,6 +180,15 @@ type MediatedHostDevice struct {
 	ExternalResourceProvider bool   `json:"externalResourceProvider,omitempty"`
 }
 
+// ResourceRequirements is a list of resource requirements for the operand workloads pods
+// +k8s:openapi-gen=true
+type OperandResourceRequirements struct {
+	// StorageWorkloads defines the resources requirements for storage workloads. It will propagate to the CDI custom
+	// resource
+	// +optional
+	StorageWorkloads *corev1.ResourceRequirements `json:"storageWorkloads,omitempty"`
```

Comment:
is it worth to set this under the workload stanza?

Reply:
I don't think we can set the same values for all the operand's workloads. It is also only supported in CDI.

Reply:
agrred